docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.9'
+
+services:
+
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./game_project:/game_project
+    command: >
+      sh -c "python3 manage.py makemigrations &&
```

Comment:
Hey, I don't think you need to do `makemigrations` here, you should leave migrations as a part of the codebase and just run `migrate` on startup

Migrations are always a part of repository in git (there's the same statement in django docs, but I didn't find it quickly)